docs/reference/environment-variables.rst

```diff
@@ -8,86 +8,74 @@
 Environment variables
 *********************
 
+.. envvar:: ARCHFLAGS
 
-.. todo::
+   This environmental variable is used for supporting architecture cross
+   compilation on macOS in a way compatible with setuptools_. It is ignored on
+   all other platforms. It can be set to ``-arch arm64`` or to ``-arch
+   x86_64`` for compiling for the aarch64 and the x86-64 architectures
```

Comment:
minor: `arm64` is the better architecture name to use for macOS specifically I think, that's what it's called in most docs/resources.